src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
```

Comment:
redundant React fragment (the same in the MovieCard component) - check usage: [doc](https://react.dev/reference/react/Fragment)